WordPress/Classes/Extensions/URL+Helpers.swift

```diff
@@ -115,6 +115,17 @@ extension URL {
             return pathExtension.lowercased() == "gif"
         }
     }
+
+    func appendHideMasterbarParameters() -> URL? {
```

Comment:
I'd personally rename this slightly, as the current name (to me) implies that it's going to modify the URL that you call it on, whereas it's actually returning a new URL. Perhaps `appendingHideMasterbarParameters` or `urlWithAppendedMasterbarParameters`?